### PR TITLE
feat: Add unhighlightTag prop

### PR DESCRIPTION
**Pull request by kaungmyatlwin**
## Description

This PR adds ability to render custom HTML tags / React Nodes instead of defaulting to `<span>` element.

## Problem
Defaulting to `<span>` is a good idea and must have, but sometimes we have edge cases where you need to pass custom React nodes in order to provide additional functionalities. Please see the Use Case below.

## Use Case
Should be able to render `<a>` tags whenever necessary or pass React components (such as `<Linkify>`: https://github.com/tasti/react-linkify) to render the links properly.

### CODE REVIEW

File: src/Highlighter.test.js [hunk 1]
```diff
@@ -372,4 +374,17 @@ describe('Highlighter', () => {
     expect(node.classList.contains('test-class')).to.equal(true)
     expect(node.dataset.testAttribute).to.equal('data attribute content')
   })
+
+  it('should use :unhighlightTag if provided', () => {
```
Comment (sergei-startsev) [inline src/Highlighter.test.js:378]: Could you also add tests to prove that `highlightTag` can be a functional/class component?

**Review by sergei-startsev — CHANGES_REQUESTED** [review]
@kaungmyatlwin Thanks for your contribution. I like the idea to have consistent `highlight*` and `unhighlight*` properties. Left a comment.

**Comment by kaungmyatlwin** [comment]
Hi @sergei-startsev , thanks for replying. I've added two test cases f8d34a1b4be84b12804933bd01537c20f7e8154f for `unhighlightTag` to test if they can be functional/class component.

Please have a look.

**Comment by sergei-startsev** [comment]
@kaungmyatlwin please exclude `package-lock.json`, `yarn.lock` from the PR.

**Review by sergei-startsev — CHANGES_REQUESTED** [review]
@kaungmyatlwin please exclude package-lock.json, yarn.lock from the PR.

**Comment by kaungmyatlwin** [comment]
@sergei-startsev Rebased against master. Can you check?

**Review by sergei-startsev — APPROVED** [review]

**Comment by sergei-startsev** [comment]
@kaungmyatlwin your changes will be included into the next release.

**Comment by JeffreyATW** [comment]
@sergei-startsev Hi there! Would you mind creating a new release with this change? Thanks!

**Comment by sergei-startsev** [comment]
@JeffreyATW check out https://www.npmjs.com/package/react-highlight-words/v/0.20.0

**Pull request merged**